src/components/basic/film/ppFilm.tsx

```diff
@@ -0,0 +1,43 @@
+// *Styles*
+import "./ppFilm.scss";
+
+import Chip from "../chip/chip";
+
+interface Props {
+  textColour?: string;
+}
+
+const fakeData = {
+  title: "The Avengers",
+  poster: "icons/poster.jpeg",
+  description:
+    "Nick Fury is compelled to launch the Avengers Initiative when Loki poses a threat to planet Earth. His squad of superheroes put their mindas together to accomplish the task.",
+  releaseYear: "2012",
+  genres: ["Action", "Superhero"],
+};
+
+const PPFilm = ({ textColour = "primary" }: Props) => {
+  return (
+    <div className="pp-film-container">
+      <img
+        src={fakeData.poster}
+        alt="movie poster"
+        className="pp-film-poster"
+      />
+      <div className="pp-film-info">
+        <p className="pp-film-title">{fakeData.title}</p>
+        <p className="pp-film-description">{fakeData.description}</p>
+        <div className="pp-film-chip-container">
+          {fakeData.genres.map((genre) => (
```

Comment:
Not sure if it shows an error on your IDE but when you loop objects, they need to be give a key. Please can you update to be:  (i is the index)

```
{fakeData.genres.map((genre, i) => (
  <Chip text={genre} key={i} />
))}
 ```